entrees/cajun-mac-n-cheese.md

```diff
@@ -0,0 +1,49 @@
+# Cajun mac-n-cheese
+
+This mac-n-cheese recipe combines Bill's innovation of incorporating spaghetti sauce into mac-n-cheese with a Google search of cajun fettucine recipes.
+
+
+## Ingredients (for 6 bowls)
+
+- 2 boxes of Kraft (or similar) mac n cheese, only one cheese packet required
+- 3 tbsp oil
+- 1/2 lb medium raw shrimp
+- 1/2 small onion, diced
+- 2 cloves garlic, minced
+- 1/2 bell pepper, diced
+- 1/8 cup sherry
+- 1/4 margarine
+- 1/4 cup shredded mozzarella
+- 1/4 cup milk
+- 1/4 cup grated parmesan
+
+### Cajun spices
+- 1.5 tsp paprika
+- 1 tsp celery salt
+- 1 tsp garlic powder
+- 1 tsp oregano
+- 1 tsp thyme
+- 1/2 tsp black pepper
+- 1/2 tsp onion powder
+- 1/2 tsp cayenne
+
+
+## Instructions
+
+1. Thaw the 1/2 lb of medium shrimp, either in the fridge overnight or by running the unopened bag under room temperature water. Drain away any melted water from the thawed shrimp.
+
+2. Start boiling enough water in a pot based on the mac-n-cheese box directions.
+
+3. Heat 3 tbsp oil in large pan.
+
+4. Dice half a small onion and half a bell pepper, and mince 2 cloves of garlic. Mix all the spices and herbs together in a small bowl.
+
+5. Saute the diced onion in the pan. After the onion begins to become transparent, add the minced garlic and diced bell pepper to the pan. Do not let the garlic burn.
+
+6. Once pot of water is boiling, add macaroni from both boxes along with a pinch of salt.
+
+7. When oil in the pan is hot, add all 1/2 lb of the raw, thawed shrimp to the pan, and mix it in with the onions and garlic. Once the shrimp begins to turn pink, pour in the 1/8 cup of sherry. Once most of the liquid is boiled off, take off the heat and set aside.
```

Comment:
I would cook the shrimp first (both sides) then remove the shrimp and cook the veggies in the pan with the brown bits left by the shrimp (good flavor). Add the sherry and only add the shrimp at almost the very end to not overcook them and maintain the browned color